composer.json

```diff
@@ -11,14 +11,14 @@
         }
     ],
     "require": {
-        "php": "^5.5|^7.0"
+        "php": "^5.5 || ^7.0"
     },
     "require-dev": {
         "php-http/httplug": "^1.0",
         "php-http/message-factory": "^1.0",
-        "puli/composer-plugin": "1.0.0-beta9",
+        "puli/composer-plugin": "1.0.0-beta9 || 1.0.0-beta10",
```

Comment:
I do not think so. It is in the docs. (My BC pr)

If we want to do this properly we should config Travis to run tests with different versions of puli.